package.json

```diff
@@ -25,7 +25,10 @@
     "chalk": "4.1.1",
     "commander": "4.1.0",
     "fast-glob": "^3.2.7",
-    "lodash": "4.17.21"
+    "husky": "^7.0.4",
+    "lint-staged": "^12.3.4",
+    "lodash": "4.17.21",
+    "prettier": "npm:wp-prettier@^2.2.1-beta-1"
```

Comment:
What's the reasoning for using a `wp-prettier` beta version here for the `prettier` dependency? That looks potentially problematic to me 🤔

Reply:
Hey @felixarntz, 

I explained that choice in [this comment](https://github.com/WordPress/performance/pull/177#issuecomment-1049756635). Pasting relevant contents here for easy reference:

> I have used [wp-prettier](https://www.npmjs.com/package/wp-prettier) instead of prettier as it's a fork of prettier which is also [used by Gutenberg](https://github.com/WordPress/gutenberg/blob/50b5bfa7de660d41a3b0e0693bf3fc9fc56836d8/package.json#L200). The changes made by the WP team is documented in its readme:
> ```
> This is a fork of Prettier that adds a new command-line option --paren-spacing which inserts many extra spaces inside parentheses, the way how projects in the WordPress ecosystem (Calypso, Gutenberg, etc.) like to format their code.
> ``` 

Basically. we needed prettier for dependency and then I checked [Gutenberg repo](https://github.com/WordPress/gutenberg/blob/50b5bfa7de660d41a3b0e0693bf3fc9fc56836d8/package.json#L200) and they were using this exact version, so I used the same one. 

I don't think the beta tag in the version should be a problem as this package is an internal fork of `prettier` maintained by WordPress. Also,  the given version, `2.2.1-beta-1`, was released 1 year ago and has been used by Gutenberg since then so I think it's safe to use.